tasks/app/views/tasks/index.html.erb

```diff
@@ -6,14 +6,15 @@
     <h1>Task List</h1>
     <div class="center">
       <% @tasks.each do |task| %>
-        <p>
-          <% if !task.completed.nil?  %>
-            <a href="/tasks/<%= task.id %>"><%= task.name %></a>
-          <% else %>
-            <a href="/tasks/<%= task.id %>" class="strikethrough"><%= task.name %></a>
-          <% end %>
-        </p>
+        <span>
+          <a href="/tasks/<%= task.id %>"<% if !task.completed.nil? %>class="strikethrough"<% end %>><%= task.name %></a>&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
Because of the three different ERB tags on this line, it might be a good candidate for using the `link_to` helper method.